modules/ztp-sno-siteconfig-config-reference.adoc

```diff
@@ -74,18 +74,15 @@ When creating the `bmh-secret` CR, use the same namespace as the `SiteConfig` CR
 The default value is `UEFI`. Use `UEFISecureBoot` to enable secure boot on the host.
 
 |`spec.clusters.nodes.rootDeviceHints`
-|Specifies the device for deployment. Identifiers that are stable across reboots are recommended, for example, `wwn: <disk_wwn>` or `deviceName: /dev/disk/by-path/<device_path>`. For a detailed list of stable identifiers, see the "About root device hints section".
+|Specifies the device for deployment. Identifiers that are stable across reboots are recommended. For example, `wwn: <disk_wwn>` or `deviceName: /dev/disk/by-path/<device_path>`. `<by-path>` values are preferred. For a detailed list of stable identifiers, see the "About root device hints" section.
```

Comment:
AFAIK, `by-path` is not a user-replaceable value, so it doesn't need angle brackets:
```suggestion
|Specifies the device for deployment. Identifiers that are stable across reboots are recommended. For example, `wwn: <disk_wwn>` or `deviceName: /dev/disk/by-path/<device_path>`. `by-path` values are preferred. For a detailed list of stable identifiers, see the "About root device hints" section.
```
or maybe better to avoid starting a sentence with a backticked item:
```suggestion
|Specifies the device for deployment. Identifiers that are stable across reboots are recommended. For example, `wwn: <disk_wwn>` or `deviceName: /dev/disk/by-path/<device_path>`. Values using a `by-path` scheme are preferred. For a detailed list of stable identifiers, see the "About root device hints" section.
```